diff -urNp -x '*.orig' shaderc-2024.0.org/CMakeLists.txt shaderc-2024.0/CMakeLists.txt
--- shaderc-2024.0.org/CMakeLists.txt	2024-04-04 01:12:49.909224636 +0200
+++ shaderc-2024.0/CMakeLists.txt	2024-04-04 01:12:49.953225227 +0200
@@ -158,6 +158,6 @@ function(define_pkg_config_file NAME LIB
   endif()
 endfunction()
 
-define_pkg_config_file(shaderc -lshaderc_shared)
+define_pkg_config_file(shaderc -lshaderc)
 define_pkg_config_file(shaderc_static "-lshaderc ${EXTRA_STATIC_PKGCONFIG_LIBS} -lshaderc_util")
 define_pkg_config_file(shaderc_combined -lshaderc_combined)
diff -urNp -x '*.orig' shaderc-2024.0.org/glslc/CMakeLists.txt shaderc-2024.0/glslc/CMakeLists.txt
--- shaderc-2024.0.org/glslc/CMakeLists.txt	2024-03-08 19:52:40.000000000 +0100
+++ shaderc-2024.0/glslc/CMakeLists.txt	2024-04-04 01:12:49.953225227 +0200
@@ -30,7 +30,7 @@ add_library(glslc STATIC
 )
 
 shaderc_default_compile_options(glslc)
-target_include_directories(glslc PUBLIC ${glslang_SOURCE_DIR})
+target_include_directories(glslc PUBLIC ${glslang_SOURCE_DIR} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../libshaderc_util/include)
 
 if (SHADERC_ENABLE_WGSL_OUTPUT)
   if (IS_DIRECTORY "${tint_SOURCE_DIR}/include")
@@ -43,16 +43,14 @@ if (SHADERC_ENABLE_WGSL_OUTPUT)
 endif(SHADERC_ENABLE_WGSL_OUTPUT)
 
 target_link_libraries(glslc PRIVATE
-  glslang SPIRV    # Glslang libraries
-  $<$<BOOL:${SHADERC_ENABLE_WGSL_OUTPUT}>:libtint>      # Tint libraries, optional
-  shaderc_util shaderc                                  # internal Shaderc libraries
+  shaderc_shared
   ${CMAKE_THREAD_LIBS_INIT})
 
 add_executable(glslc_exe src/main.cc)
 shaderc_default_compile_options(glslc_exe)
-target_include_directories(glslc_exe PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/.. ${spirv-tools_SOURCE_DIR}/include)
+target_include_directories(glslc_exe PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/.. ${CMAKE_CURRENT_SOURCE_DIR}/../libshaderc_util/include ${CMAKE_CURRENT_SOURCE_DIR}/../libshaderc/include ${spirv-tools_SOURCE_DIR}/include)
 set_target_properties(glslc_exe PROPERTIES OUTPUT_NAME glslc)
-target_link_libraries(glslc_exe PRIVATE glslc shaderc_util shaderc)
+target_link_libraries(glslc_exe PRIVATE glslc shaderc_shared shaderc_util SPIRV-Tools)
 add_dependencies(glslc_exe build-version)
 
 shaderc_add_tests(
diff -urNp -x '*.orig' shaderc-2024.0.org/libshaderc/CMakeLists.txt shaderc-2024.0/libshaderc/CMakeLists.txt
--- shaderc-2024.0.org/libshaderc/CMakeLists.txt	2024-03-08 19:52:40.000000000 +0100
+++ shaderc-2024.0/libshaderc/CMakeLists.txt	2024-04-04 01:12:49.953225227 +0200
@@ -41,7 +41,7 @@ target_compile_definitions(shaderc_share
     PRIVATE SHADERC_IMPLEMENTATION
     PUBLIC SHADERC_SHAREDLIB
 )
-set_target_properties(shaderc_shared PROPERTIES SOVERSION 1)
+set_target_properties(shaderc_shared PROPERTIES OUTPUT_NAME shaderc SOVERSION 1)
 
 if(SHADERC_ENABLE_INSTALL)
   install(
