--- shaderc-2021.0/libshaderc/CMakeLists.txt.orig	2021-04-28 20:30:58.667969817 +0200
+++ shaderc-2021.0/libshaderc/CMakeLists.txt	2021-04-28 20:31:43.873665942 +0200
@@ -41,7 +41,7 @@ target_compile_definitions(shaderc_share
     PRIVATE SHADERC_IMPLEMENTATION
     PUBLIC SHADERC_SHAREDLIB
 )
-set_target_properties(shaderc_shared PROPERTIES SOVERSION 1)
+set_target_properties(shaderc_shared PROPERTIES OUTPUT_NAME shaderc SOVERSION 1)
 
 if(SHADERC_ENABLE_INSTALL)
   install(
--- shaderc-2021.0/glslc/CMakeLists.txt.orig	2021-04-28 18:26:01.925013098 +0200
+++ shaderc-2021.0/glslc/CMakeLists.txt	2021-04-28 18:34:04.012401406 +0200
@@ -30,7 +30,7 @@
 )
 
 shaderc_default_compile_options(glslc)
-target_include_directories(glslc PUBLIC ${glslang_SOURCE_DIR})
+target_include_directories(glslc PUBLIC ${glslang_SOURCE_DIR} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../libshaderc_util/include)
 
 if (SHADERC_ENABLE_WGSL_OUTPUT)
   if (IS_DIRECTORY "${tint_SOURCE_DIR}/include")
@@ -42,17 +42,14 @@
   add_definitions(-DSHADERC_ENABLE_WGSL_OUTPUT=1)
 endif(SHADERC_ENABLE_WGSL_OUTPUT)
 
-target_link_libraries(glslc PRIVATE
-  glslang OSDependent OGLCompiler HLSL glslang SPIRV    # Glslang libraries
-  $<$<BOOL:${SHADERC_ENABLE_WGSL_OUTPUT}>:libtint>      # Tint libraries, optional
-  shaderc_util shaderc                                  # internal Shaderc libraries
+target_link_libraries(glslc PRIVATE shaderc_shared
   ${CMAKE_THREAD_LIBS_INIT})
 
 add_executable(glslc_exe src/main.cc)
 shaderc_default_compile_options(glslc_exe)
-target_include_directories(glslc_exe PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/.. ${spirv-tools_SOURCE_DIR}/include)
+target_include_directories(glslc_exe PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/.. ${CMAKE_CURRENT_SOURCE_DIR}/../libshaderc_util/include ${CMAKE_CURRENT_SOURCE_DIR}/../libshaderc/include ${spirv-tools_SOURCE_DIR}/include)
 set_target_properties(glslc_exe PROPERTIES OUTPUT_NAME glslc)
-target_link_libraries(glslc_exe PRIVATE glslc shaderc_util shaderc)
+target_link_libraries(glslc_exe PRIVATE glslc shaderc_shared shaderc_util SPIRV-Tools)
 add_dependencies(glslc_exe build-version)
 
 shaderc_add_tests(
--- shaderc-2021.0/CMakeLists.txt.orig	2021-05-02 21:59:17.590975179 +0200
+++ shaderc-2021.0/CMakeLists.txt	2021-05-02 22:00:32.646136904 +0200
@@ -150,6 +150,6 @@
   endif()
 endfunction()
 
-define_pkg_config_file(shaderc -lshaderc_shared)
+define_pkg_config_file(shaderc -lshaderc)
 define_pkg_config_file(shaderc_static "-lshaderc ${EXTRA_STATIC_PKGCONFIG_LIBS} -lshaderc_util")
 define_pkg_config_file(shaderc_combined -lshaderc_combined)
